ExadelTimeTrackingSystem/ExadelTimeTrackingSystem.Data/Repositories/ProjectRepository.cs

```diff
@@ -35,5 +35,12 @@ public Task<string> GetNameAsync(Guid id)
             var filter = filterBuilder.Eq(p => p.Id, id);
             return GetCollection<Project>().Find(filter).Project(p => p.Name).SingleOrDefaultAsync();
         }
+
+        public System.Threading.Tasks.Task UpdateAsync(Project project)
```

Comment:
put Update method to base repository, and if it's used in some other repositories, remove them as well, one implementation in base class can used for all repos